src/collective/liveblog/upgrades/v1002/__init__.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding:utf-8 -*-
+from collective.liveblog.config import PROJECTNAME
+from collective.liveblog.interfaces import ILiveblog
+from plone import api
+from plone.app.upgrade.utils import loadMigrationProfile
+
+import logging
+
+
+logger = logging.getLogger(PROJECTNAME)
+
+
+def remove_workflow(context):
```

Comment:
I followed @pbauer suggestion to use [ftw.upgrade](https://github.com/4teamwork/ftw.upgrade)